src/contribution/projects-explore.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * window.c.contribution.ProjectsExplore component
+ * A root component to show projects according to user defined filters
+ *
+ * Example:
+ * To mount this component just create a DOM element like:
+ * <div data-mithril="ProjectsExplore">
+ */
+window.c.contribution.ProjectsExplore = ((m, c, h, _) => {
+    return {
+
+        controller: () => {
+            const filters = m.postgrest.filtersVM,
+                  nearMe = filters({near_me: 'eq', state: 'eq'}).near_me(true),
+                  expiring = filters({expires_at: 'lte', state: 'eq'}).state('online').expires_at(moment().add(14, 'days').format('YYYY-MM-DD')),
+                  recents = filters({online_date: 'gte', state: 'eq'}).state('online').online_date(moment().subtract(5, 'days').format('YYYY-MM-DD')),
+                  recommended = filters({recommended: 'eq', state: 'eq'}).recommended('true').state('online'),
+                  online = filters({state: 'eq'}).state('online'),
+                  byCategory = filters({category_id: 'eq'}),
+                  projects = m.postgrest.paginationVM(c.models.project, 'project_id.desc'),
+                  successful = filters({state: 'eq'}).state('successful'),
+                  lCategories = m.postgrest.loader(
+                      c.models.category.getPageOptions(
+                          filters({}).order({name: 'asc'}).parameters()
+                      )
+                      , m.postgrest.request
+                      , true
+                  );
+
+            const filtersMap = {
+                recommended: {
+                    title: 'Recomendados',
+                    filter: recommended
+                },
+                online: {
+                    title: 'No ar',
+                    filter: online
+                },
+                expiring: {
+                    title: 'Reta final',
+                    filter: expiring
+                },
+                successful: {
+                    title: 'Bem-sucedidos',
+                    filter: successful
+                },
+                recents: {
+                    title: 'Recentes',
+                    filter: recents
+                },
+                near_me: {
+                    title: 'Próximos a mim',
+                    filter: nearMe
+                }
+            };
+
+            const vm = {
+                categoryCollection: m.prop([]),
+                title: m.prop(),
+                category: m.prop(),
+
+                loadProjects: (title, filter, category) => {
+                    vm.title(title);
+                    vm.category(category);
+                    projects.firstPage(filter.parameters());
+                    vm.toggleCategories.toggle();
+                },
+
+                loadRoute: () => {
```

Comment:
Yes, but I'm still not sure on how to use then for this situation. We can definitely chat about this.